pyproject.toml

```diff
@@ -74,9 +74,9 @@ dependencies = [
     "scipy>=1.14",
     "Shapely>=2",
     "splink>=4",
-    "sphinx>=8,<8.1",
+    "sphinx>=8",
     "sphinx-autoapi>=3",
-    "sphinx-issues>=1.2",
+    "sphinx-issues>=5",
```

Comment:
Love a 4 major version jump.

Reply:
I know. I asked to be made a maintainer in January of 2022 and... 🦗🏏🦗